src/main/java/virtual_pet/VirtualPet.java

```diff
@@ -0,0 +1,46 @@
+package virtual_pet;
+public class VirtualPet {
+
+    public String petName;
+    public int hunger;
+    public int thirst;
+    public int energy;
+
+    public VirtualPet(String petName, int hunger, int thirst, int energy) {
+
+        this.petName = petName;
+        this.hunger = hunger;
+        this.thirst = thirst;
+        this.energy = energy;
+    }
+    public void changeHunger(int h){
+        hunger += h;}
+    public void changeThirst(int t){
+        thirst += t;
+    }
+    public void changeEnergy(int e){
+        energy += e;
+    }
+    public String getPetName(){
+        return petName;
+    }
+    public int getHunger(){
+        return hunger;}
+    public int getThirst(){
+        return thirst;}
+    public int getEnergy(){
+        return energy;
+    }
+    public void setHunger(int hunger){
+        this.hunger = hunger;
+    }
+    public void setThirst(int thirst){
+        this.thirst = thirst;
+    }
+
+    public static void tick(VirtualPet pet) {
```

Comment:
Method should not be static. and should not have a parameter variable.  Your tick() could have simply could have been complete with just the following lines:
public void tick(){
    this.hunger -= 1;
    this.thirst -= 1; 
}

The tick() you have here is too complex and overdoing things as well as decreasing performance and efficiency of the program.